Vikta-API-testing-sample/src/test/java/com/test/api/imageitem/ImageItemTest.java

```diff
@@ -432,8 +432,12 @@ private static Stream<Arguments> nonAcceptableDataForAdvancedSearch() {
         newTagsList.add("MO");
         return Stream.of(
                 Arguments.of(19.33, 1.25, 5, 0, newTagsList),
```

Comment:
According to the documentation a couple of test cases shouldn't work but it turns out that it works. I think that this could be a mistake in Vikta code